index.html

```diff
@@ -349,13 +372,19 @@
         <pre class="idl">
         dictionary PerformanceObserverInit {
           required sequence&lt;DOMString&gt; entryTypes;
+          boolean buffered = false;
         };
         </pre>
         <dl>
           <dt><dfn>entryTypes</dfn></dt>
           <dd>A list of entry names to be observed. The list MUST NOT be empty
           and types not recognized by the user agent MUST be ignored.</dd>
         </dl>
+        <dl>
+          <dt><dfn>buffered</dfn></dt>
+          <dd>A flag to indicate whether buffered entries should be queued into
+          observer's buffer.</dd>
```

Comment:
"queued into the `<a>performance entry buffer</a>`", right? Calling "queue" should always queue into observer's buffer.